src/SSAO.js

```diff
@@ -255,6 +255,12 @@ x3dom.SSAO.blur = function ( stateManager, gl, scene, ssaoTexture, depthTexture,
 
     stateManager.useProgram( sp );
 
+    // Mask ssao w/ fog, no need for fogColor
+    // ALWAYS pass FOG uniforms into shader or scenes w/out fog will barf
```

Comment:
Could you check for fog here and in the SSAO shader (properties.FOG, see below) ?

Reply:
ideally yes, however (unlike shapes & shadows) properties are not being created in the ssao chain, and I was hesitant to shoehorn them into x3dom.Utils.wrapProgram.